web/templates/eula.html.tmpl

```diff
@@ -0,0 +1,39 @@
+{{ define "content" }}
+    <div class="mb-4">
+        <div class="row">
+            <div class="col-sm-8">
+                <h1 class='display-4 lead'>License agreement</h1>
+                <p class='subheadline'>An open cloud-native web console improving the
+                  life of SAP Applications administrators</p>
+                <hr/>
+                <p>Trento is an open source (Apache License 2.0) component that is supported as part of SUSE Linux Enterprise Server for SAP applications and therefore falls under the standard SUSE Terms and Conditions that can be reviewed at <a href="https://www.suse.com/products/terms_and_conditions.pdf">https://www.suse.com/products/terms_and_conditions.pdf</a>, SUSE EULA is available at <a href="https://www.suse.com/licensing/eula/#server">https://www.suse.com/licensing/eula/#server</a>.</p>
+
+                <p>By using Trento via a registered SUSE Linux Enterprise Server for SAP you agree to the Terms and Conditions also for Trento.</p>
+
+
+                <p>Trento collects the following KPIs on your environment and transmits it to SUSE:<p>
+                <ul>
+                  <li>SUSE Customer Center ID</li>
+                  <li>Number of registered hosts in Trento</li>
+                  <li>For each of those hosts:
+                    <ul>
+                      <li>SLES version</li>
+                      <li>Number of sockets and total count of cores per host</li>
+                      <li>Available memory per host</li>
+                      <li>Hosting platform (Cloud service provider, Hypervisor or bare Metal)</li>
```

Comment:
We need to disclaim the collection of the following per-hosts data:
- List of running SAP instances
- Statistics about Trento configuration checks